src/cypress/integration/pages/TradePage.js

```diff
@@ -0,0 +1,79 @@
+class TradePage {
```

Comment:
I like the `TradePage` concept. General comment about the selectors, we need to make some less brittle and some more specific. I've made a few comments, but please apply those comments to all the selectors.

Reply:
fixed in latest commit.